code/controlconfig/controlsconfigcommon.cpp

```diff
@@ -1399,14 +1392,14 @@ int control_config_common_write_tbl(bool overwrite = false) {
 			buf_str = "NONE";
 		}
 
-		cfputs(("$Bind Name: " + item.text + "\n").c_str(), cfile);
+		puts(("$Bind Name: " + item.text + "\n").c_str(), cfile);
```

Comment:
Use our cfile versions of functions such as `puts`, as they use their own allocator and can handle other SCP containers.  Don't directly use c std stuff for file reading and writing.